.github/workflows/authenticate_test.yml

```diff
@@ -25,9 +25,13 @@ jobs:
         ports:
           - 9042:9042
         options: --health-cmd "cqlsh --username cassandra --password cassandra --debug" --health-interval 5s --health-retries 30
+      env:
+        DOCKERHUB_USERNAME: ${{ secrets.DOCKERHUB_USERNAME }}
+        DOCKERHUB_TOKEN: ${{ secrets.DOCKERHUB_TOKEN }}
```

Comment:
```suggestion
        credentials:
          username: ${{ secrets.DOCKERHUB_USERNAME }}
          password: ${{ secrets.DOCKERHUB_TOKEN }}
```

Reply:
Will token work as password?

Reply:
Yes, https://docs.docker.com/security/for-developers/access-tokens/#use-an-access-token